Driverless AI/disparate-impact-analysis-tutorial/disparate-impact-analysis-tutorial.md

```diff
@@ -0,0 +1,458 @@
+
+![shapley](assets/shapley.jpg)
+
+5. You can also check the *Partial Dependence Plot* and see the probability of defaulting increases when PAY_0 is two months late. 
+
+![partial-dependence-plot-of-PAY_0](assets/partial-dependence-plot-of-PAY_0.jpg)
+
+Now that we know that being two months late on PAY_0 is terrible and knowing that the average mean score prediction is *0.24060*, what will occur if we were to set all customers to have PAY_0=2? Will the average mean score prediction increase or decrease? 
+
+To set all customers PAY_0 to 2, please consider the following steps: 
+
+  1. Click on top of the PAY_0 variable. 
+
+![change-pay-0](assets/change-pay-0.jpg)
+
+  2. A box will appear to make sure the absolute radio button is selected. Set the *Absolute* to 2. After, click *Set*.
+
+![pay-0-equal-to-2](assets/pay-0-equal-to-2.jpg)
+
+  4. Click the *RESCORE* button. 
+
+![rescore](assets/rescore.jpg)
+
+5. We can check “Current Working Set Score” on the right to see a summation of what occurred. In this run, we see that by switching all to PAY_0=2 that we over doubled the average mean, implying that our model would reject significantly more of the credits because the perceived probability of default was much higher. Our current score is now *0.46028* from a prior score of *0.24060*. Consequently, the absolute change is of *0.21968* an increase of *91.31%* change.
+
+![delta](assets/delta.jpg)
+
+
+6. Let's take this one step further and try another adjustment. Let's adjust PAY_AMT2 to 80% of what it originally was i.e., Let's see what happens to the model when we make our entire population only pay 80% of what they did. As you may notice, the variable *PAY_AMT2* is not on the table, but don't worry, we can add it by following these quick steps: 
+
+    1. Click on the *Plus* icon, and a box will appear. 
+
+    ![add-option](assets/add-option.jpg)
+
+    2. Check the *PAY_AMT2* variable and click *SET*
+
+    3. Right after, similar to how we change the value of *PAY_0*, we will click on the *PAY_AMT2* variable. 
+
+    4. Select the *Percentage* radio button and set the *Percentage* to 80. Click *SET*. 
+
+    ![pay2-80-percent](assets/pay2-80-percent.jpg)
+
+    5. Click the *RESCORE* button. 
+
+  Interestingly enough, when we finish rescoring and review results, we discover that this adjustment had virtually no effect whatsoever. In this case, the absolute change when modifying *PAY_AMT2* is a *0.00074* increase. It is a crucial consideration in Machine Learning to be aware that there will be many situations where you might think a variable would be important where it is not and vice versa.
+
+![no-change](assets/no-change.jpg)
+
+6. Let's inspect this from a different angle, but before, let's restore the original *Sensitivity Analysis* results. Click the reset button located on the bottom right corner of the *Sensitivity Analysis* graph.  
+
+![reset](assets/reset.jpg)
+
```

Comment:
Do you have anything that we coul add for the Deeper Diver and Resources?

Reply:
@Castro38 solved